openmdao_book/other_useful_docs/auto_ivc_api_translation.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Maybe not a big deal but why does this example show a way to get the value of x but not in the other version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Neither one should. Good catch</p>